src/dials/command_line/generate_mask.py

```diff
@@ -153,19 +182,43 @@ def run(args: List[str] = None, phil: phil.scope = phil_scope) -> None:
     )
 
     # Parse the command line arguments
-    params, options = parser.parse_args(args=args, show_diff_phil=True)
+    params, options, unhandled = parser.parse_args(
+        args=args, show_diff_phil=True, return_unhandled=True
+    )
     experiments = flatten_experiments(params.input.experiments)
 
     # Configure logging
     dials.util.log.config(verbosity=options.verbose, logfile=params.output.log)
 
+    # Read in any starting masks
+    remains_unhandled = []
+    starting_masks = []
+    for arg in unhandled:
```

Comment:
Just because it will unpickle does not make it a mask? Need to check that the unpicked object is the right type?